things/api.py

```diff
@@ -0,0 +1,141 @@
+# -*- coding: utf-8 -*-
+
+"""
+Module implementing Things API
+"""
+
+
+from .database import Database
+
+
+def tasks(type="task", status="open", area=None, project=None, heading=None, **kwargs):
+    """
+    Read tasks into a list of dicts.
+
+    Parameters
+    ----------
+    type : {'task', 'project', None}, optional, default 'task'
+        Only return a specific type of Task:
+
+        'task':     Default. Task within a project or a standalone task;
+                    can link to an area, tags, and a checklist.
+        'project':  a supertask; can link to an area, tags, (sub)tasks,
```

Comment:
If project is a "supertask", isn't a task also a "supertask" of checklist items?

Reply:
I'm glad you're mentioning this. The terminology used internally in the database can be somewhat confusing.

Intuitively speaking, a task is something you "check off". Whereas an area acts like a container or folder. So when I say "supertask" I'm pointing to something you can check off which consists of tasks.

As for individual checklist items, one definitely can check them off. However, in the Things database, they are their own table. So strictly speaking they aren't Things-tasks, but rather "checklist items". Maybe one way to view it, is that checklist items act more like a part of the notes of a task. They don't have tags, due dates, notes, or other such features.

That said, feel free to change the documentation if you find it confusing,